pkg/loader/load.go

```diff
@@ -484,7 +492,9 @@ func newBatchManager(s *loaderImpl) *batchManager {
 		fExecDDL:             s.execDDL,
 		fDDLSuccessCallback: func(txn *Txn) {
 			s.markSuccess(txn)
```

Comment:
mark success but fail to refresh, does it have any effect？

Reply:
mark success used for save checkpoint, if refreshTableInfo failed, only influence the dmls after this ddl.  I think it's ok to markSuccess here.

Reply:
It sent the transaction to the channel returned by `Successes()`, which is used in `Syncer.handleSuccess` and is mainly used to parse for timestamp and be saved as checkpoints.
Since at this point the DDL is already synced to the downstream database and `refreshTableInfo` only affect in memory status of drainer, it should be OK.